.github/workflows/reusable-e2e.yml

```diff
@@ -233,8 +233,8 @@ jobs:
       - name: Configure liveness credentials
         uses: aws-actions/configure-aws-credentials@e1e17a757e536f70e52b5a12b2e8d1d1c60e04ef # v2.0.0 https://github.com/aws-actions/configure-aws-credentials/commit/e1e17a757e536f70e52b5a12b2e8d1d1c60e04ef
         with:
-          aws-region: us-west-2
-          role-to-assume: ${{ secrets.LIVENESS_E2E_ROLE_ARN }}
+          aws-region: us-east-2
```

Comment:
I just want to lament that it took way too long to find this spot

Reply:
😭

Reply:
😭

Reply:
Is this correct? We dont have Liveness launched in us-east-2

Reply:
Our amplify app is hosted in us-east-2 but calls the liveness service in us-east-1 because liveness is not launched there